src/Debug/ProfilerSection.php

```diff
@@ -98,7 +106,14 @@ public function getName(): string
     public function getDuration(): int
     {
         $end = $this->end ?? (int)(microtime(true) * 1000);
-        return $end - $this->start;
+        $duration = $end - $this->start;
+
+        // Remove paused time from the total runtime.
+        foreach ($this->pauses as $pause) {
+            $duration -= $pause[1] - $pause[0];
```

Comment:
It looks like it is possible to have a "paused" section at the end of execution (due to error or due to a missing call to `resume()`). It should be handled here to prevent a `undefined index` error.